src/client/cmd/start.cpp

```diff
@@ -59,25 +61,19 @@ mp::ReturnCode cmd::Start::run(mp::ArgParser* parser)
                                "instance.\n");
                 }
             }
-            else
```

Comment:
Since this else is deleted, I would prefer to see the corresponding `if` incorporated into the parent `if`, to reduce one nesting level

Reply:
Yeah, I should be able to tighten this up some.

Reply:
Tightened this up some.  Let me know what you think.